exercises/standard_library_types/iterators5.rs

```diff
@@ -0,0 +1,61 @@
+// iterators5.rs
+
+pub fn fibonacci(i: u64) -> u64 {
+    // Complete this function to return the ith fibonacci number
+    // Do not use:
+    // - return
+    // For extra fun don't use:
+    // - imperative style loops (for, while)
+    // - let
+    // For the most fun don't use:
+    // - recursion
+    // Scroll down for hints.
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn fibonacci_of_1() {
+        assert_eq!(1, fibonacci(1));
+    }
+    #[test]
+    fn fibonacci_of_2() {
+        assert_eq!(2, fibonacci(2));
+    }
+
+    #[test]
+    fn fibonacci_up_to_10() {
+        assert_eq!(
+            "[1, 1, 2, 3, 5, 8, 13, 21, 34, 55]",
+            format!("{:?}", (0..10).map(fibonacci).collect::<Vec<u64>>())
+        );
+    }
+}
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Why is there so much whitespace here?

Reply:
It's due to the old format of hints (they were down below) I fixed it in my branch but before publishing a PR, I prefers to give some time to the original author.